### PR TITLE
Feature/format lint

### PR DESCRIPTION
**Pull request by alewin**
- add `prepare` command  used by `husky` hooks
- add generic `fix` and `check` format and lint commands
- add `husky` lint e format checks

### CODE REVIEW

File: .husky/pre-push
Comment (SamSalvatico) [inline .husky/pre-push:1]: Wouldn't be better to have it on pre-commit so to have clean commits?

**Review by SamSalvatico — COMMENTED** [review]
Please give a look to what @edge33 did on

File: .husky/pre-push
Comment (alewin) [reply .husky/pre-push:1]: yes make sense, there are only few files and biome is fast so it doesn't really annoy

**Review by SamSalvatico — APPROVED** [review]

**Review by edge33 — APPROVED** [review]
LGTM

**Pull request merged**